src/main/java/org/prebid/server/bidder/beachfront/BeachfrontBidder.java

```diff
@@ -258,23 +240,34 @@ private static User makeUserForVideoRequest(User user) {
      */
     private static BeachfrontBannerRequest makeBannerRequest(BidRequest bidRequest, List<Imp> imps,
                                                              List<String> errors) {
-        final BeachfrontBannerRequest.BeachfrontBannerRequestBuilder beachfrontRequestsBuilder =
+        final BeachfrontBannerRequest.BeachfrontBannerRequestBuilder beachfrontBannerRequestBuilder =
```

Comment:
Such local vars can be called like `requestsBuilder` because no reason here for too much details. We have no other request builder here. But up to you, just a suggestion.